src/components/studies/launch/LaunchAlerts.tsx

```diff
@@ -293,7 +304,14 @@ const StudyAlertComponent: React.FunctionComponent<
           </div>
           {error.isDismissable && (
             <Box className={classes.reviewIgnoreButtons}>
-              <Button href={SECTIONS[sectionIndex].path}>Review</Button>
+              <Button
+                href={
+                  SECTIONS[sectionIndex].path === 'customize'
```

Comment:
I think it's better to handle it more generally. The other sections don't care about the 'from' parameter, but potentially they might so just doing     <Button href={`${SECTIONS[sectionIndex].path}?from=launch`} ... makes sense